dot/config_test.go

```diff
@@ -462,8 +462,8 @@ func TestLogConfig_String(t *testing.T) {
 				BlockProducerLvl:  log.Warn,
 				FinalityGadgetLvl: log.Error,
 			},
-			want: "core: DBUG, digest: INFO, sync: WARN, network: EROR, rpc: CRIT, state: DBUG, runtime: INFO, " +
-				"block producer: WARN, finality gadget: EROR",
+			want: "core: DEBUG   , digest: INFO    , sync: WARN    , network: ERROR   , rpc: CRITICAL, " +
+				"state: DEBUG   , runtime: INFO    , block producer: WARN    , finality gadget: ERROR   ",
```

Comment:
Oh-oh... maybe add a method such as `IndentedString() string` for the one adding spaces? And keep `String() string` without the spaces? 🤔 
Since that config with spaces is just megaweird

Reply:
> Another thought, maybe let's just have an unexported method `func (l Level) getSpaceSuffixes() int` with a switch which return the number of spaces to add after when logging? Otherwise the whole logging code/config strings look rather weird.

@qdm12 Im a bit confused, how does this fix the strings looking weird? I can see how it cleans up the expected string, but if we want padding, for tests that test logs wouldn't the spaces still be present in the string (like above test)?

Reply:
So you would return the level as before (sorta) with `String()` (`INFO`, ..., `CRITICAL`)

And then have:

```go
func (l Level) formatWithTrailingSpaces() (formatted string) {
  switch l {
  case Info:
    return "info    "
  // ...
  case Critical:
    return "critical "
  }
}
```

and use `level.formatWithTrailingSpaces()` in the `log` function in the logger package, right before writing to the io.Writer.

Reply:
ah okay yes that sounds good thank you